src/Filecoin/Transaction.h

```diff
@@ -0,0 +1,56 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "Address.h"
+#include "../Cbor.h"
+#include "../uint256.h"
```

Comment:
we don't need `uint256_t` anymore? types in proto file are `uint64_t` now

Reply:
The values are still 256-bit in the Filecoin protocol though, and `uint256.h` is needed by the cpp file.
Wouldn't it be nicer to stay close to the specs?
We just changed the .proto interface to make integration into the apps easier I suppose.

Reply:
in this case, let's revert to `bytes` in proto? uint64 might be overflow

Reply:
I think @vikmeup wanted uint64 in the proto interface because Filecoin only has 8 decimal places.
That should leave the max re-presentable amount `184,467,440,737.09551615`, so about 184 billion. Filecoin's max supply is only two billion though. https://coinlist.co/assets/index/filecoin_index/Filecoin-Sale-Economics-e3f703f8cd5f644aecd7ae3860ce932064ce014dd60de115d67ff1e9047ffa8e.pdf

Now that I think about it, dropping precision to 64-bit in all code actually makes sense. :)
What do you think @hewigovens ?

Reply:
yes, make all uint64_t

Reply:
done sir